docs/terraform.md

```diff
@@ -1,362 +1,40 @@
 # Terraform
 
-Terraform is an infrastructure of code tool which we use to deploy our application to AWS.  This section's goal is to document our specific use of terraform in our project and how we use it to deploy our application.  We will start by explaining how and why we use terraform in Dawson, and later you can get a more detailed understanding of terraform after we have explained how terraform is setup in our project.
+# How Dawson Uses Terraform
```

Comment:
This looks great Zach, thank you! 🎉 I think what you've got here is exactly the kind of thing I've been longing for -- enough contextual info to get a grasp on why things are the way they are without being overly verbose, and more importantly, without re-documenting someone else's tool/service.

The only thing I might add is a link straight to the terraform docs in your `Resources` section, but that's a take-it-or-leave-it comment since finding terraform docs is easy enough to do with a quick google search.